assets/js/modules/adsense/components/module/ModuleTopEarningPagesWidget/Table.js

```diff
@@ -33,12 +33,30 @@ import ReportTable from '../../../../../components/ReportTable';
 import TableOverflowContainer from '../../../../../components/TableOverflowContainer';
 import Link from '../../../../../components/Link';
 import { MODULES_ANALYTICS, DATE_RANGE_OFFSET } from '../../../../analytics/datastore/constants';
+import { MODULES_ADSENSE } from '../../../datastore/constants';
 import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { getCurrencyFormat } from '../../../util/currency';
 import { generateDateRangeArgs } from '../../../../analytics/util/report-date-range-args';
 import { numFmt } from '../../../../../util';
 const { useSelect } = Data;
 
 export default function Table( { report } ) {
+	const {
+		currencyFormat,
+	} = useSelect( ( select ) => {
```

Comment:
Nit-pick, but no need to wrap this in an object here since it's the only returned value.